samples/benchmarks/coremark/boards/nrf54h20dk_nrf54h20_cpuflpr.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+CONFIG_COREMARK_ITERATIONS=10000
+
+# Disable the UART console Kconfig to make Kconfig configuration with the DTS configuration.
+CONFIG_UART_CONSOLE=n
+
+# Kconfigs required for the STM standalone logging, imported from the nordic-log-stm snippet.
+CONFIG_TEST_LOGGING_DEFAULTS=n
+CONFIG_LOG_FRONTEND=y
+CONFIG_LOG_FRONTEND_ONLY=y
+CONFIG_LOG_FRONTEND_STMESP=y
+CONFIG_LOG_FRONTEND_STMESP_FSC=y
+
+# Disable the NCS boot banner - the application core is responsible for printing the boot banner.
+CONFIG_NCS_BOOT_BANNER=n
+CONFIG_BOOT_BANNER=n
+
+# Reduce speed optimizations to fit the sample into the TCM of the FLPR core.
+# Removed the following options from the default configuration (prj.conf):
+# -funroll-loops
+CONFIG_COMPILER_OPT="-O3 -fno-lto -fno-pie -fno-pic -ffunction-sections -fdata-sections"
```

Comment:
`-fno-pie -fno-pic` are default in Zephyr, sop why are they added here ?
`-ffunction-sections -fdata-sections"` are default in Zephyr, sop why are they added here ?

Reply:
This is a copy-paste of the original options from prj.conf (minus -funroll-loops) that you added. @zycz, why did you add them?

Reply:
I got them from SAG experiments.

We also wanted to mention them explicitly to show them in output logs - we use this variable here:

#define COMPILER_FLAGS CONFIG_COMPILER_OPT " + see compiler flags added by Zephyr"

but I guess we don't have to add it as we already mention that we  use default zephyr flags

Reply:
okay, so I will clean up these flags

Reply:
done